api/src/routes/flashcard.rs

```diff
@@ -88,3 +88,47 @@ pub async fn get_flashcard(
 
     HttpResponse::Ok().json(flashcards)
 }
+
+exportable! {
+    pub struct FlashcardFromIDGet {
+        flashcard_id: i32
+    }
+}
+
+exportable! {
+    pub struct FlashcardFromIDResp {
+        id: i32,
+        folder_id: i32,
+        term: String,
+        definition: String
+    }
+}
+
+#[get("/flashcard/fromid")]
```

Comment:
Thinking for v2 we should have
- `GET /folder/flashcards` to get flashcards in the folder
- `GET /flashcard` for this (from id)

Thoughts?

Reply:
That sounds good. Let me know what you think of the idea of caching flashcard info on the review selection screen as I mention in a couple of comments - I'm not sure if this is either easy or impossible using tanstack's react-query, or if we'd have to resort to storage/cookies.